### PR TITLE
(pouchdb/couchdb-harness#19) - auth fixes

### PR DESCRIPTION
**Pull request by marten-de-vries**

### CODE REVIEW

File: lib/routes/special-test-auth.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    var namePass = header.slice('X-Couch-Test-Auth '.length);
+    var name = namePass.split(':')[0];
+    var pass = namePass.split(':')[1];
+
+    var expectedPw = {
+      "Jan Lehnardt": "apple",
+      "Christopher Lenz": "dog food",
+      "Noah Slater": "biggiesmalls endian",
+      "Chris Anderson": "mp3",
+      "Damien Katz": "pecan pie"
```
Comment (nolanlawson) [inline lib/routes/special-test-auth.js]: Now I can finally log in to all their accounts. :)

**Comment by nolanlawson** [comment]
+1 when green

**Comment by marten-de-vries** [comment]
couchdb-harness isn't green yet. Didn't run the complete test (only the parts I was working on) locally which shows now.

**Comment by marten-de-vries** [comment]
Ok, the last commit fixed the problem (mostly thanks to a pouchdb-auth update) :)

**Pull request merged**